conduct.md

```diff
@@ -32,8 +33,20 @@ As a member of the community, you agree to the following:
 
 - Keep unstructured critique to a minimum. We encourage sharing ideas and perspectives, so please ensure that your feedback is constructive and relevant. If you have solid ideas you want to experiment with, make a fork and see how it works.
 - Avoid aggressive and micro-aggressive behavior, such as unconstructive criticism, providing corrections that do not improve the conversation (sometimes referred to as "well actually"s), repeatedly interrupting or talking over someone else, feigning surprise at someone’s lack of knowledge or awareness about a topic, or subtle prejudice (for example, comments like “That’s so easy my grandmother could do it.”). For more examples of this kind of behavior, [see the Recurse Center's user manual](https://www.recurse.com/manual#sec-environment).
```

Comment:
This wasn't changed, but by using "That' s so easy my grandmother could do it" as an example of micro-aggressive behavior against the person you're talking to, aren't we demonstrating age discrimination?  Maybe we want to eat our own dog food here and not insult grandmothers?

Reply:
I thought the point was that it insults both the listener _and_ grandmothers. But if you didn't read it that way, perhaps there's a clarity problem that should be addressed. (But not in this PR, let's stay focused on the changes Tim is proposing.)

Reply:
Thanks. I agree that further changes should be suggested in new PRs.